### PR TITLE
[cfgmgr] clear loopback and vrf in kernel if not warmstart

### PR DESCRIPTION
**Pull request by tylerlinp**
**What I did**
Clear loopback interfaces in kernel at intfmgrd starting except warmstart.
Clear vrf interfaces in kernel at vrfmgrd starting except warmstart.

**Why I did it**
After loopback moved to intfmgrd, the behavior is changed. 
[issue #3814](https://github.com/Azure/sonic-buildimage/issues/3814)

**How I verified it**
tests

**Details if related**

### CODE REVIEW

File: cfgmgr/intfmgr.cpp [hunk 1]
```diff
@@ -107,6 +112,27 @@ void IntfMgr::delLoopbackIntf(const string &alias)
...
+    if (ret)
+    {
+        SWSS_LOG_DEBUG("Command '%s' failed with rc %d", cmd.str().c_str(), ret);
+        return;
+    }
+
+    auto aliases = tokenize(res, '\n');
+    for (string &alias : aliases)
+    {
+        delLoopbackIntf(alias);
```
Comment (prsunny) [inline cfgmgr/intfmgr.cpp]: Can you add a LOG here?

**Comment by prsunny** [comment]
retest this please

File: cfgmgr/intfmgr.cpp (hunk 1 above)
Comment (tylerlinp) [reply cfgmgr/intfmgr.cpp]: OK.

**Comment by tylerlinp** [comment]
retest this please

**Pull request merged**